runtime/drivers/olap.go

```diff
@@ -2,13 +2,15 @@ package drivers
 
 import (
 	"context"
+	"github.com/rilldata/rill/runtime/connectors/sources"
 
 	"github.com/jmoiron/sqlx"
 )
 
 // OLAPStore is implemented by drivers that are capable of storing, transforming and serving analytical queries
 type OLAPStore interface {
 	Execute(ctx context.Context, stmt *Statement) (*sqlx.Rows, error)
+	Ingest(ctx context.Context, source sources.Source, parsedOptions any) (bool, *sqlx.Rows, error)
```

Comment:
1. The `parsedOptions any` here feels a little hacky. I think we should omit it, and then overrides can access the source's properties directly, or import the specific connectors and call their config-parsing functions.
2. Since we return an error here, we don't need the bool – we can just add a `ErrUnsupportedConnector` or something like that, which drivers should return if they don't have an overwrite.

Paired with earlier comments, this would become:

```go
var ErrUnsupportedConnector = errors.New("driver: connector not supported")

type OLAPStore interface {
  // ...
  Ingest(ctx context.Context, source connectors.Source) error
}
```